### PR TITLE
Copyedit store plugins list in persistence.rst

### PR DESCRIPTION
**Pull request by dymil**
# Summary of changes
This is just a documentation change to make the list of store plugins grammatical and parallel.

# Checklist

- [X] Checked that there aren't other open pull requests for
  the same change.

### CODE REVIEW

File: docs/persistence.rst [hunk 1]
```diff
@@ -70,9 +70,9 @@ Additional store plugins
 
 More store implementations are available in RDFLib extension projects: 
 
- * `rdflib-sqlalchemy <https://github.com/RDFLib/rdflib-sqlalchemy>`_, which supports stored on a wide-variety of RDBMs backends, 
- * `rdflib-leveldb <https://github.com/RDFLib/rdflib-leveldb>`_ - a store on to of Google's `LevelDB <https://code.google.com/p/leveldb/>`_ key-value store. 
- * `rdflib-kyotocabinet <https://github.com/RDFLib/rdflib-kyotocabinet>`_ - a store on to of the `Kyoto Cabinet <http://fallabs.com/kyotocabinet/>`_ key-value store. 
+ * `rdflib-sqlalchemy <https://github.com/RDFLib/rdflib-sqlalchemy>`_ – a store which supports a wide-variety of RDBMS backends, 
+ * `rdflib-leveldb <https://github.com/RDFLib/rdflib-leveldb>`_ – a store onto Google's `LevelDB <https://code.google.com/p/leveldb/>`_ key-value store.
```
Comment (aucampia) [inline docs/persistence.rst:74]: I think it is clear with `on top of`

```suggestion
 * `rdflib-leveldb <https://github.com/RDFLib/rdflib-leveldb>`_ – a store on top of Google's `LevelDB <https://code.google.com/p/leveldb/>`_ key-value store. 
```

File: docs/persistence.rst [hunk 2]
```diff
@@ -70,9 +70,9 @@ Additional store plugins
 
 More store implementations are available in RDFLib extension projects: 
 
- * `rdflib-sqlalchemy <https://github.com/RDFLib/rdflib-sqlalchemy>`_, which supports stored on a wide-variety of RDBMs backends, 
- * `rdflib-leveldb <https://github.com/RDFLib/rdflib-leveldb>`_ - a store on to of Google's `LevelDB <https://code.google.com/p/leveldb/>`_ key-value store. 
- * `rdflib-kyotocabinet <https://github.com/RDFLib/rdflib-kyotocabinet>`_ - a store on to of the `Kyoto Cabinet <http://fallabs.com/kyotocabinet/>`_ key-value store. 
+ * `rdflib-sqlalchemy <https://github.com/RDFLib/rdflib-sqlalchemy>`_ – a store which supports a wide-variety of RDBMS backends, 
+ * `rdflib-leveldb <https://github.com/RDFLib/rdflib-leveldb>`_ – a store onto Google's `LevelDB <https://code.google.com/p/leveldb/>`_ key-value store. 
+ * `rdflib-kyotocabinet <https://github.com/RDFLib/rdflib-kyotocabinet>`_ – a store onto the `Kyoto Cabinet <http://fallabs.com/kyotocabinet/>`_ key-value store.
```
Comment (aucampia) [inline docs/persistence.rst:75]:
```suggestion
 * `rdflib-kyotocabinet <https://github.com/RDFLib/rdflib-kyotocabinet>`_ – a store on top of the `Kyoto Cabinet <http://fallabs.com/kyotocabinet/>`_ key-value store. 
```

**Review by aucampia — COMMENTED** [review]
Thanks for the PR @dymil - I added two suggestions.

**Comment by dymil** [comment]
I agree; I'd just picked which of the original phrasings was better.

**Review by nicholascar — APPROVED** [review]
Thanks again for the copy edits!

**Pull request merged**